File: app/src/main/java/com/soya/launcher/ui/fragment/GuideWifiListFragment.java
package com.soya.launcher.ui.fragment;

import android.os.Bundle;

public class GuideWifiListFragment extends AbsWifiListFragment {
    public static GuideWifiListFragment newInstance() {

        Bundle args = new Bundle();

        GuideWifiListFragment fragment = new GuideWifiListFragment();
        fragment.setArguments(args);
        return fragment;
    }
}

File: app/src/main/java/com/soya/launcher/ui/fragment/GuideWifiListFragment.kt
package com.soya.launcher.ui.fragment

import android.os.Bundle
import com.shudong.lib_base.base.BaseViewModel
import com.soya.launcher.databinding.FragmentWifiListBinding

object GuideWifiListFragment : AbsWifiListFragment<FragmentWifiListBinding, BaseViewModel>() {
    @JvmStatic
    fun newInstance(): GuideWifiListFragment {
        val args = Bundle()

        val fragment = GuideWifiListFragment
        fragment.arguments = args
        return fragment
    }
}
